appconfiguration/azure-spring-cloud-feature-management/dynamic-features-sample/src/main/java/com/azure/spring/dynamicfeatures/ApplicationProperties.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.azure.spring.dynamicfeatures;
+
+import java.util.Map;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import com.azure.spring.cloud.feature.manager.IDynamicFeatureProperties;
+import com.azure.spring.dynamicfeatures.models.ShoppingCart;
+
+@Configuration
+@ConfigurationProperties(prefix = "config")
```

Comment:
`config` is not a good name prefix. How about using prefix like `application.dynamic.features`